### PR TITLE
Show winning market results in "My Positions" view

### PR DESCRIPTION
**Pull request by mikec**
and add a computed `winningMarket` property for each decision to state, to use in all views

### CODE REVIEW

File: app/reducers/computed/getWinningMarket.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import decisionStatuses from '../../constants/decisionStatuses'
+import decisionMarketTypes from '../../constants/decisionMarketTypes'
+
+const getWinningMarket = decision => {
+  if (decision.status !== decisionStatuses.OPEN) {
```
Comment (emaG3m) [inline app/reducers/computed/getWinningMarket.js]: why does the status need to be OPEN to get a winning market index?

File: app/reducers/computed/getWinningMarket.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    if (decision.resolved) {
+      return decision.passed ? decisionMarketTypes.YES : decisionMarketTypes.NO
+    } else if (decision.yesMarketPrice && decision.noMarketPrice) {
+      return decision.yesMarketPrice > decision.noMarketPrice ?
```
Comment (emaG3m) [inline app/reducers/computed/getWinningMarket.js]: guessing this is `yesLongPrice` and `noLongPrice`

File: app/reducers/decisionMarkets.js [hunk 3]
```diff
@@ -130,9 +136,17 @@ const decisionMarkets = (state = [], action) => {
           }
         }
       })
-    default:
-      return state
+      break
   }
+  return addComputedProps(returnState)
```
Comment (emaG3m) [inline app/reducers/decisionMarkets.js]: confused as to the placement of this...when does it get added?

File: app/components/decisionTables/DecisionResultCell.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import MarketNameStyled from './MarketNameStyled'
+
+const DecisionResultCell = ({ decisionResult }) => (
+  <MarketNameStyled type={decisionResult} />
```
Comment (emaG3m) [inline app/components/decisionTables/DecisionResultCell.js]: wondering why this needs to be wrapped here. seems extraneous?

File: app/reducers/computed/getWinningMarket.js (hunk 1 above)
Comment (mikec) [reply app/reducers/computed/getWinningMarket.js]: good question .. probably makes sense to just set it, and add logic to check OPEN vs. RESOLVED elsewhere

File: app/reducers/computed/getWinningMarket.js (hunk 2 above)
Comment (mikec) [reply app/reducers/computed/getWinningMarket.js]: yes, but these are the average prices from `StandardMarketWithPriceLogger.sol` ... maybe we should be explicit about that

File: app/reducers/decisionMarkets.js (hunk 3 above)
Comment (mikec) [reply app/reducers/decisionMarkets.js]: this just runs through the whole state (which in this reducer is the array of decision market data) and adds computed properties (so far, only the `winningMarket` property). So it's doing this on every dispatched action.

File: app/components/decisionTables/DecisionResultCell.js (hunk 4 above)
Comment (mikec) [reply app/components/decisionTables/DecisionResultCell.js]: really just for consistency .. all of the components used in the tables are `<Something>Cell` components. Not sure I love this pattern though...

File: app/reducers/decisionMarkets.js (hunk 3 above)
Comment (emaG3m) [reply app/reducers/decisionMarkets.js]: despite the other return statements?

File: app/components/decisionTables/DecisionResultCell.js (hunk 4 above)
Comment (emaG3m) [reply app/components/decisionTables/DecisionResultCell.js]: kk

**Comment by mikec** [comment]
@emaG3m I changed that status logic and refactored some of the naming for state props

**Pull request merged**